integration/simulation/simulation.go

```diff
@@ -227,7 +227,7 @@ func (s *Simulation) deployObscuroERC20s() {
 
 			deployContractTx := types.DynamicFeeTx{
 				Nonce:     NextNonce(s.ctx, s.RPCHandles, owner),
-				Gas:       2_000_000,
+				Gas:       5_000_000,
```

Comment:
What is the reason for increasing gas here, while decreasing it in `func PrefundWallets`?
Were previous estimates way off and we are adjusting to make them more optimised?

Reply:
Just making the gas limit equal everywhere pretty much.
Also, I'm not sure why we're using the DynamicFee tx in that particular tx and didn't want to over tweak it.